src/intl/en/page-eth.json

```
@@ -15,11 +15,13 @@
   "page-eth-flexible-amounts": "Available in flexible amounts",
   "page-eth-flexible-amounts-desc": "ETH is divisible up to 18 decimal places so you don't have to buy 1 whole ETH. You can buy fractions at a time – as little as 0.000000000000000001 ETH if you want.",
   "page-eth-fuels": "ETH fuels and secures Ethereum",
-  "page-eth-fuels-desc": "ETH is the lifeblood of Ethereum. When you send ETH or use an Ethereum application, you'll pay a small fee in ETH to use the Ethereum network. This fee is an incentive for a miner to process and verify what you're trying to do.",
-  "page-eth-fuels-desc-2": "Miners are like the record-keepers of Ethereum – they check and prove that no one is cheating. Miners who do this work are also rewarded with small amounts of newly-issued ETH.",
+  "page-eth-fuels-desc": "ETH is the lifeblood of Ethereum. When you send ETH or use an Ethereum application, you'll pay a fee in ETH to use the Ethereum network. Part of the fee is required by the network, and part is an incentive for a block producer to process and include what you're trying to do.",
```

```suggestion
  "page-eth-fuels-desc": "ETH is the lifeblood of Ethereum. When you send ETH or use an Ethereum application, you'll pay a fee in ETH to use the Ethereum network. Part of the fee is required by the network (<a to=\"/glossary/#contract-account\">base fee</a>), and part is an incentive for a block producer (priority fee) to process and include what you're trying to do.",
```

Just a thought about adding terminology here and linking to the glossary terms. Should probably add something for the priority fee though if we do this.

---

```suggestion
"page-eth-fuels-desc": "ETH is the lifeblood of Ethereum. When you send ETH or use an Ethereum application, you'll pay a fee in ETH to use the Ethereum network. Part of the fee is required by the network (<a to=\"/glossary/#base-fee\">base fee</a>), and part is an incentive for a block producer (priority fee) to process and include what you're trying to do.",
```